uint8                   mode              # mode to execute
int16                   value
